ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -23,6 +23,18 @@ import java.time.{Duration, Instant}
   "Errors raised by or forwarded by the Ledger API."
 )
 object LedgerApiErrors extends LedgerApiErrorGroup {
+  object UnsupportedFeature
+      extends ErrorCode(
+        id = "UNSUPPORTED_FEATURE",
+        ErrorCategory.InternalUnsupportedOperation,
+      ) {
+
+    case class Reject(unsupportedFeature: String)(implicit errorLogger: ContextualizedErrorLogger)
+        extends LoggingTransactionErrorImpl(
+          cause = s"The request exercised an unsupported ledger feature: $unsupportedFeature"
```

Comment:
```suggestion
          cause = s"The request exercised an unsupported operation: $unsupportedFeature"
```

Make it wider so we don't need to introduce more kinds of unsupported operation errors.